go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/diskfs/go-diskfs
 
-go 1.21
+go 1.23
```

Comment:
Was there any reason to drop the support for Go 1.22?

Reply:
Not particularly. It kind of went along with other flows. We can step it back. Open a PR and make sure it passes lint and tests, and happy to merge it in.

Reply:
- https://github.com/diskfs/go-diskfs/pull/262